src/api/curriculumApi.ts

```diff
@@ -102,7 +102,7 @@ function htmlToText(html: string) {
 }
 
 function filterTextsForLanguage(texts: Text[], language: string) {
-  const isoCode = isoLanguageMapping[language.substring(0, 2)] || 'default';
+  const isoCode = isoLanguageMapping[language?.split('-')?.[0]] || 'default';
```

Comment:
Språk hentes enten fra parameter eller kontekst. Kontekstspråket er enten nb, nn eller en, fra språkvelgeren som sendes i header accept-language. I graphql playground brukes språket til nettleseren og for meg er det 'nb-NO,nb;q=0.9,en-GB;q=0.8,en;q=0.7,no;q=0.6,nn;q=0.5,en-US;q=0.4'. Denne varianten håndtere begge formata, og også språkkoder på tre bokstaver, som sma.